get_latest_ppa_deb.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
This is nice, should we instead try to integrate some of this logic with the [`sync-upstream.yaml` CI](https://github.com/snapcrafters/obs-studio/blob/candidate/.github/workflows/sync-upstream.yml)?